cue/schemas/plugins/bar/panel.cue

```diff
@@ -0,0 +1,16 @@
+package bar
```

Comment:
this file tree looks heavy from a far point of view.
Maybe we should talk about it, I'm certainly missing something here.

Reply:
I can remove one imbrication layer at the top actually, didnt think about it when I changed the file archi